lib/UnixConfigLoader.php

```diff
@@ -9,6 +9,23 @@
 
 class UnixConfigLoader implements ConfigLoader
 {
+    const MAX_NAMESERVERS = 3;
+    const MAX_DNS_SEARCH = 6;
+
+    const MAX_TIMEOUT = 30 * 1000;
+    const MAX_ATTEMPTS = 5;
+    const MAX_NDOTS = 15;
+
+    const DEFAULT_TIMEOUT = 5 * 1000;
```

Comment:
Any reason to increase the default timeout from 3000 to 5000 milliseconds?

Reply:
@kelunik that's what resolv.conf assumes as the default value, IMO we should assume the same value here